Ducky-Chickens/bangazon-site
# Joe db

# Description
I added the build-db.js file and set up the requires and excute the bulkCreate functions for each table.
This PR has NONE of the models in it, so you cant really test it. 

## Related Ticket(s)
Database

## Problem to Solve
Give us database

## Proposed Changes
have database

## Expected Behavior
make database

## Steps to Test Solution

1. Since this PR doesnt have the models in it, it wont be able to be tested until those are written. 
2. You should double check the naming conventions in the build-db.js file to make sure they are correct

## Testing


## Documentation

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
